File: InjuredAndroid/app/src/main/java/b3nac/injuredandroid/b25lActivity.java
package b3nac.injuredandroid;

import android.content.Context;
import android.content.SharedPreferences;
import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;

public class b25lActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_b25l);
        FlagsOverview.flagTwoButtonColor = true;

        SharedPreferences settings = getSharedPreferences("b3nac.injuredandroid", Context.MODE_PRIVATE);
        SharedPreferences.Editor editor = settings.edit();
        editor.putBoolean("flagTwoButtonColor", true).commit();
    }
}

File: InjuredAndroid/app/src/main/java/b3nac/injuredandroid/b25lActivity.kt
package b3nac.injuredandroid

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

class b25lActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_b25l)
        SecureSharedPrefs.setContext(this)

        FlagsOverview().flagTwoButtonColor = true

        SecureSharedPrefs().editBoolean(this, "flagTwoButtonColor", true)
    }
}
